kernel/pipedev.c

```diff
@@ -154,8 +173,8 @@ static int _pd_pipe2(myst_pipedev_t* pipedev, myst_pipe_t* pipe[2], int flags)
         /* Set the state */
         shared->state = STATE_WR_ENABLED;
 
-        /* Set the non-blocking flag */
-        shared->flags = flags;
+        /* Set the pipe id (for debugging) */
```

Comment:
Should this operation be under `ENABLE_TRACE` compiling switch?

Reply:
Yes, I think that is cleaner. I was leaving it for gdb inspection but it is probably better to remove it.

Reply:
Fixed